XPCBackupService/Services/BackupRealm.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  BackupRealm.swift
+//  XPCBackupService
+//
+//  Created by Richard Ascanio on 3/8/24.
+//
+
+import Foundation
+import RealmSwift
+
+struct BackupRealm {
+    static let shared = BackupRealm()
+
+    private init() {}
+
+    func getRealm() throws -> Realm {
+        do {
+            let config = Realm.Configuration(schemaVersion: 2)
+            Realm.Configuration.defaultConfiguration = config
+            return try Realm(fileURL: ConfigLoader.realmURL)
+        } catch {
+            throw BackupUploadError.CannotCreateRealm
+        }
+    }
+
+    func addSyncedNodeToDB(_ node: SyncedNode) throws {
```

Comment:
don't use the concept of DB here, as it can be confusing, `BackupRealm.shared.addSyncedNode` is contextual enough